api/envoy/extensions/filters/udp/udp_proxy/v3/udp_proxy.proto

```diff
@@ -50,7 +54,15 @@ message UdpProxyConfig {
     option (validate.required) = true;
 
     // The upstream cluster to connect to.
-    string cluster = 2 [(validate.rules).string = {min_len: 1}];
+    string cluster = 2 [
+      deprecated = true,
+      (validate.rules).string = {min_len: 1},
+      (envoy.annotations.deprecated_at_minor_version) = "3.0"
+    ];
```

Comment:
I would like to not deprecate `cluster` since generic matcher API is working in progress and the current syntax is too complicated for routing to a single cluster. What is your opinion @mattklein123 ?

Reply:
I would like to hear from @snowp on this. I think there should be some way of doing an any match which should not be very verbose.